contracts/scripts/contracts.sh

```
@@ -24,6 +24,8 @@ export TRADE_SYSTEMS=$(get_contract_address "eternum-trade_systems")
 
 export RESOURCE_SYSTEMS=$(get_contract_address "eternum-resource_systems")
 
+export RESOURCE_BRIDGE_SYSTEMS=$(get_contract_address "eternum-resource_bridge_systems")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)